pkg/arvo/sys/vane/ames.hoon

```diff
@@ -3655,6 +3650,16 @@
           |=  =bone
           ^+  peer-core
           abet:(call:(abed:mi:peer-core bone) %flub ~)
+        ::
+        ++  check-clog
+          |=  [=bone =ship]
+          ^+  peer-core
+          =/  =message-pump-state  (~(got by snd.peer-state) bone)
+          ?:  ?&  !=(0 (end 0 bone))
+                  (gth ~(wyt in unsent-messages.message-pump-state) msg.cong.ames-state)
```

Comment:
Do we need the byte count check as well? I could put it here but it kinda feels unnecessary.